oneview_redfish_toolkit/mockups/redfish/ZoneWithoutDrives.json

```diff
@@ -8,9 +8,78 @@
     },
     "Links": {
         "ResourceBlocks": [
+            {
+                "@odata.id": "/redfish/v1/CompositionService/ResourceBlocks/30373737-3237-4D32-3230-313531354752"
+            },
+            {
+                "@odata.id": "/redfish/v1/CompositionService/ResourceBlocks/30373737-3237-4D32-3230-313531394752"
+            },
+            {

```

Comment:
No problem for now, but maybe we could use less data to put in mockups, to avoid extensives mockups with similar data.

Reply:
Fixed on last commit. Now, we're using just 4 sh to validate zone unit tests.